person.rb

```diff
@@ -1,16 +1,16 @@
 require_relative './nameable'
 
 class Person < Nameable
-  def initialize(age, name = 'unknown', parent_permission: true)
-    @id = rand(1..10_000)
+  def initialize(id, age, name = 'unknown', parent_permission: true)
+    @id = id
     super()
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = object.new
   end
 
-  attr_accessor :name, :age
-  attr_reader :id
+  attr_accessor :name, :age, :id
```

Comment:
- [ ] You seem to have forgotten to add the rentals . This should be ```attr_accessor :name, :age, :rentals```

Reply:
Done! Thanks